src/tables.ts

```diff
@@ -933,6 +935,11 @@ const durationCodes: Record<string, any> = {
         getWidth: () => Tables.SLASH_NOTEHEAD_WIDTH,
         position: 'B/4',
       },
+      p: {
+        // Parenthesis drawn by notehead.draw()
+        code_head: 'noteheadDoubleWhole',
+        getWidth: () => Tables.PARENTHESIS_NOTEHEAD_WIDTH,
```

Comment:
This fixed width works for slashed notes, but I don't think for parenthesis (across all fonts and notehead types) -- you'll need to get the current width, and add the width of two parens + some spacing.